content/projects/github-api-consume/_index.md

```diff
@@ -50,3 +50,9 @@ Please be sure to follow the standard naming conventions for your language.
 ## Resources
 
 - [An introduction to curl using GitHub's API](https://gist.github.com/tazjel/8735770).
+
+### Instructions for reviewers
+- Ensure that the function works with any github User and repository name, meaning that the function should not only work with one User/repo name. 
+- Ensure that the function prints and does not return the output.
```

Comment:
@kaleem99 I think you might have missed my comment here https://github.com/Umuzi-org/ACN-syllabus/pull/276#discussion_r757443220

Reply:
In JS it should print, in Python it should return.

In JS it's a bit weird because it's async.